### PR TITLE
Organize and document simulation.tensor

### PR DESCRIPTION
**Pull request by mpharrigan**
- Rename `qualtran.simulation.quimb_sim` to `qualtran.simulation.tensor`
 - Slight refactoring to make a closer correspondence between methods and functions
 - Document!

### CODE REVIEW

**Comment by mpharrigan** [comment]
it's very confusing that `black` reports that all files are unchanged when flynt or isort do their thing

File: qualtran/simulation/tensor.ipynb [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    "tn, fix = cbloq_to_quimb(cbloq)\n",
+    "tn.draw(show_inds=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f18e1f2d",
+   "metadata": {},
+   "source": [
+    "The entire quite of Quimb tools are now available."
```
Comment (fdmalone) [inline qualtran/simulation/tensor.ipynb:150]: suite I imagine

**Review by fdmalone — APPROVED** [review]

**Comment by fdmalone** [comment]
Nice notebook.

File: qualtran/simulation/tensor.ipynb (hunk 1 above)
Comment (mpharrigan) [reply qualtran/simulation/tensor.ipynb:150]: quite of suimb :)

nice catch

**Pull request merged**